uint32 lane_id
geometry_msgs/Point[] points
